dashboards/k8s/k8s-cluster-prometheus.json

```diff
@@ -0,0 +1,318 @@
+              "mode": "COLOR"
+            },
+            "dataSets": [
+              {
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "node_memory_MemTotal_bytes - node_memory_MemFree_bytes"
+                }
+              }
+            ],
+            "timeshiftDuration": "0s",
+            "yAxis": {
+              "label": "y1Axis",
+              "scale": "LINEAR"
+            }
+          }
+        },
+        "width": 6,
+        "xPos": 6,
+        "yPos": 3
+      },
+      {
+        "height": 4,
+        "widget": {
+          "timeSeriesTable": {
+            "dataSets": [
+              {
+                "tableDisplayOptions": {},
+                "timeSeriesQuery": {
+                  "prometheusQuery": "kube_persistentvolume_capacity_bytes"
+                }
+              }
+            ],
+            "metricVisualization": "NUMBER"
+          },
+          "title": "Persistent Volume Provisioned Capacity "
+        },
+        "width": 12,
+        "xPos": 0,
+        "yPos": 9
+      },
+      {
+        "height": 2,
+        "widget": {
+          "scorecard": {
+            "sparkChartView": {
+              "sparkChartType": "SPARK_LINE"
+            },
+            "thresholds": [
+              {
+                "color": "YELLOW",
+                "direction": "ABOVE",
+                "label": "",
+                "value": 0
+              }
+            ],
+            "timeSeriesQuery": {
+              "prometheusQuery": "sum(kube_persistentvolume_status_phase{phase=\"Available\"})"
+            }
+          },
+          "title": "PV Unbound"
+        },
+        "width": 2,
+        "xPos": 6,
+        "yPos": 7
+      },
+      {
+        "height": 2,
+        "widget": {
+          "scorecard": {
+            "sparkChartView": {
+              "sparkChartType": "SPARK_LINE"
+            },
+            "thresholds": [
+              {
+                "color": "YELLOW",
+                "direction": "ABOVE",
+                "label": "",
+                "value": 0
+              }
+            ],
+            "timeSeriesQuery": {
+              "prometheusQuery": "sum(kube_persistentvolume_status_phase{phase=\"Pending\"})"
+            }
+          },
+          "title": "PV Pending"
+        },
+        "width": 2,
+        "xPos": 8,
+        "yPos": 7
+      },
+      {
+        "height": 2,
+        "widget": {
+          "scorecard": {
+            "sparkChartView": {
+              "sparkChartType": "SPARK_LINE"
+            },
+            "thresholds": [
+              {
+                "color": "RED",
+                "direction": "ABOVE",
+                "label": "",
+                "value": 0
+              },
+              {
+                "color": "YELLOW",
+                "direction": "ABOVE",
+                "label": "",
+                "value": 0
+              }
```

Comment:
These have the same threshold, should there be different values, or can one be removed?